thorgame/ro315/Code review by waa94.py

```diff
@@ -0,0 +1,93 @@
+# ### Power of Thor - Episode 1
```

Comment:
Delete this file from your PR.

Reply:
File deleted and change commited to the ro315-1 branch